Source/Other/Extensions/StringExtension.swift

```diff
@@ -112,35 +111,29 @@ extension String {
     }
 
     public func dateStringFromUnixTimestamp() -> NSString? {
-        os_log("string [%@]", log: OSLog.default, type: .error, self)
 
         guard
+            self.length < 12, // 2121-02-17 is 4769274709 - 10 chars. 3121 is 11 chars
```

Comment:
why 2121, are you hundred years ahead? 😂

Reply:
as requested by @Jason-Morcos - I personally think it's a bit too long ... but future proof :) (excluding mysql `UNIX_TIMESTAMP()`)